boards/sensebox_samd21/board.c

```diff
@@ -21,6 +21,25 @@
 #include "cpu.h"
 #include "board.h"
 #include "periph/gpio.h"
+#include "sdcard_spi_params.h"
+#include "mtd_sdcard.h"
+
+ /* this is provided by the sdcard_spi driver
+ * see sys/auto_init/storage/auto_init_sdcard_spi.c */
+extern sdcard_spi_t sdcard_spi_devs[sizeof(sdcard_spi_params) /
+                                    sizeof(sdcard_spi_params[0])];
+mtd_sdcard_t sensebox_sd_dev = {
+    .base = {
+        .driver = &mtd_sdcard_driver,
+        .page_size = 512,
```

Comment:
Does the page_size/pages_per_sector/sector_count attributes depend on the SD card used ? Then I think (not sure though) this should be overridable from the board config using macro.

Reply:
Thanks, moved to macros